tools/accuracy_checker/accuracy_checker/metrics/detection.py

```diff
@@ -683,22 +702,71 @@ def evaluate(self, annotations, predictions):
         accuracy = true_positive / (true_positive + false_positive)
         return [accuracy]
 
-    @staticmethod
-    def calc_iou(gt_box, dt_box):
-        # Convert detected face rectangle to integer point form
-        gt_box = list(map(lambda x: int(round(x, 0)), gt_box))
-        dt_box = list(map(lambda x: int(round(x, 0)), dt_box))
 
-        # Calculate overlapping width and height of two boxes
-        inter_width = min(gt_box[2], dt_box[2]) - max(gt_box[0], dt_box[0])
-        inter_height = min(gt_box[3], dt_box[3]) - max(gt_box[1], dt_box[1])
+class AVAHeadAccuracy(FullDatasetEvaluationMetric):
 
-        if inter_width <= 0 or inter_height <= 0:
-            return None
+    __provider__ = 'ava_head_accuracy'
+    annotation_types = (DetectionAnnotation, )
+    prediction_types = (DetectionPrediction, )
 
-        intersect_area = inter_width * inter_height
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'overlap': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.40, optional=True,
+                description='Specifies the IOU threshold to consider as a true positive candidate face.'
+            ),
+            'relative_size': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.25, optional=True,
+                description='Specifies the size of detected face candidate\'s area in proportion to the size '
+                'of ground truth\'s face size. This value is set to filter candidates that have high IOU '
+                'but have a relatively smaller face size than ground truth face size.'
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.overlap = self.get_value_from_config('overlap')
+        self.relative_size = self.get_value_from_config('relative_size')
 
-        gt_area = (gt_box[2] - gt_box[0]) * (gt_box[3] - gt_box[1])
-        dt_area = (dt_box[2] - dt_box[0]) * (dt_box[3] - dt_box[1])
+    def submit_all(self, annotations, predictions):
```

Comment:
it is standart submil_all realization, please remove this method overriding

Reply:
Reflected the feedback in the new patch. Thank you.